Core/Src/main.c

```diff
@@ -90,63 +81,52 @@ static void onboot_commands(void);
 
 /* Private user code ---------------------------------------------------------*/
 /* USER CODE BEGIN 0 */
-uint8_t state = receiving;
 uint8_t RX_Data = 0xFF;
 /* USER CODE END 0 */
 
-void init_filesystem() {
-	NAND_SPI_Init(&hspi2);
-	NANDfs_init();
-}
-
-
 /**
- * @brief  The application entry point.
- * @retval int
- */
-int main(void) {
-    /* USER CODE BEGIN 1 */
-    /* USER CODE END 1 */
-
-    /* MCU Configuration--------------------------------------------------------*/
-
-    /* Reset of all peripherals, Initializes the Flash interface and the Systick. */
-    HAL_Init();
-
-    /* USER CODE BEGIN Init */
-
-    /* USER CODE END Init */
-
-    /* Configure the system clock */
-    SystemClock_Config();
-
-    /* USER CODE BEGIN SysInit */
-
-    /* USER CODE END SysInit */
-
-    /* Initialize all configured peripherals */
-    MX_GPIO_Init();
-    MX_I2C1_Init();
-    MX_I2C2_Init();
-    MX_SPI1_Init();
-    MX_SPI2_Init();
-    MX_USART1_UART_Init();
-
-    /* USER CODE BEGIN 2 */
-    NAND_SPI_Init(&hspi2);
-    char cmd[64];
-    char buf[64];
-    char *ptr = cmd;
-    onboot_commands();
-    init_filesystem();
-    /* USER CODE END 2 */
-
-    /* Infinite loop */
-    /* USER CODE BEGIN WHILE */
+  * @brief  The application entry point.
+  * @retval int
+  */
+int main(void)

```

Comment:
It rewrote this file with 2 spaces instead of 4. If we can't make the ioc comply with 4 spaces, let's switch clang-format to use 2 spaces to keep it consistent and minimize whitespace changes